README.md

```diff
@@ -1,130 +1,89 @@
-# Project Zeppelin / GDG DevFest 2014 site template
+# DevOpsDays India - Bengaluru Chapter - Website #
 
-### About
-Project Zeppelin allows you to setup awesome GDG DevFest site in 5 minutes.
+## About ##
 
-Project is built on top of [Jekyll](http://jekyllrb.com/) - simple, blog-aware, static site generator. Jekyll also happens to be the engine behind GitHub Pages, which means you can use Jekyll to host your website from GitHub’s servers for free. [Learn more about Jekyll](http://jekyllrb.com/).
+This project powers the [DevOpsDays India Website](https://devopsdaysindia.org/) (Bengaluru chapter). It uses the Project Zeppelin / GDG DevFest 2014 site template.
```

Comment:
Maybe you could leave a link to Jekyll as well in this line?

Reply:
Updated with new commit - [7d0d365](https://github.com/ashwinimanoj/devopsdays/pull/1/commits/7d0d36522f1ab3802ea40a16eefde3a0f551ee86)